astropy/io/misc/asdf/connect.py

```diff
@@ -0,0 +1,47 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+# This file connects ASDF to the astropy.table.Table class
+
+import functools
+
+import asdf
+
+from astropy.io import registry as io_registry
+from astropy.table import Table
+
+
+def read_table(filename, data_key=None, find_table=None, **kwargs):
+
```

Comment:
Needs docstring.  Hopefully soon with #8255 or variant this will be visible to the user looking for help on ASDF format-specific keywords.